starknet_devnet/util.py

```diff
@@ -124,10 +165,35 @@ def parse_args():
         action='store_true',
         help="Disables deploy tx hash calculation"
     )
+    parser.add_argument(
+        "--accounts",
+        type=int,
+        help=f"Specify the number of accounts to be predeployed; defaults to {DEFAULT_ACCOUNTS}",
+        default=DEFAULT_ACCOUNTS
+    )
+    parser.add_argument(
+        "--initial-balance", "-e",
```

Comment:
Why `-e`?

Reply:
Was following this: https://github.com/trufflesuite/ganache